api/server/controllers/assistants/chatV1.js

```diff
@@ -60,28 +59,18 @@ const chatV1 = async (req, res) => {
     parentMessageId: _parentId = Constants.NO_PARENT,
   } = req.body;
 
-  /** @type {Partial<TAssistantEndpoint>} */
-  const assistantsConfig = req.app.locals?.[endpoint];
-
-  if (assistantsConfig) {
-    const { supportedIds, excludedIds } = assistantsConfig;
+  req.query.endpoint = endpoint;
```

Comment:
Not happy with this line. I struggled to get the correct request object to pass into fetchAssistants(). Maybe there is another way to get the assistants?